internal/output/renderers/bulletlist.go

```diff
@@ -0,0 +1,89 @@
+package renderers
+
+import (
+	"strings"
+
+	"github.com/ActiveState/cli/internal/colorize"
+	"github.com/ActiveState/cli/internal/multilog"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/termutils"
+)
+
+type bulletList struct {
+	prefix  string
+	items   []string
+	bullets []string
+}
+
+// BulletTree outputs a list like:
+//
+// ├─ one
+// ├─ two
+// │ wrapped
+// └─ three
+var BulletTree = []string{output.TreeMid, output.TreeMid, output.TreeLink + " ", output.TreeEnd}
```

Comment:
Since we're using presets anyway, can we make this typed so it's easier to understand the code while reading?

eg.

```
type Bullets {
  Start string
  Mid string
  Link string
  End string
}
```